tsconfig.json

```diff
@@ -4,7 +4,8 @@
     "module": "commonjs",
     "outDir": "build",
     "strict": true,
-    "esModuleInterop": true
+    "esModuleInterop": true,
+    "importsNotUsedAsValues": "error"
```

Comment:
Better if the linter could fix these, refs https://github.com/typescript-eslint/typescript-eslint/issues/1436#issuecomment-589504369.